test/functional/completed_transaction_controller_test.rb

```diff
@@ -22,4 +24,24 @@ class CompletedTransactionControllerTest < ActionController::TestCase
       end
     end
   end
+
+  context "A/B testing" do
+    setup do
+      content_store_has_example_item("/done/vehicle-tax", schema: "completed_transaction")
+      content_store_has_example_item("/done/weee", schema: "completed_transaction")
+    end
+
+    %w[A B C D E F G H Z].each do |variant|
+      should "show the #{variant} variant of the electric car AB test on applicable pages" do
+        with_variant ElectricCarABTest: variant do
+          get :show, params: { slug: "done/vehicle-tax" }
```

Comment:
Is there a way to test that the response has been modified for the AB test here? I saw below you used `assert_response_not_modified_for_ab_test` which I hadn't seen before :)

Reply:
It's a little abstracted (and not particularly well named), but `with_variant` wraps up some assertions on this: https://github.com/alphagov/govuk_ab_testing/blob/master/lib/govuk_ab_testing/abstract_helpers.rb#L7-L20

Reply:
Ah cool, thanks! 👍